collection-spec/collection-spec.md

```diff
@@ -19,7 +19,7 @@ STAC Collections are meant to be compatible with *OGC API - Features* Collection
 | --------------- | ------------------------------------------------ | ------------------------------------------------ |
 | stac_version    | string                                           | **REQUIRED.** The STAC version the Collection implements. STAC versions can be mixed, but please keep the [recommended best practices](../best-practices.md#mixing-stac-versions) in mind. |
 | stac_extensions | \[string]                                        | A list of extension identifiers the Collection implements. |
-| id              | string                                           | **REQUIRED.** Identifier for the collection that is unique across the provider. |
+| id              | string                                           | **REQUIRED.** Identifier for the collection that is unique across the provider, ideally globally unique. |
```

Comment:
Agree after conversation in working session; suggested change removes globally unique as the ideal case.

```suggestion
| id              | string                                           | **REQUIRED.** Identifier for the collection that is unique across the provider. |
```